packages/lisk-transactions/src/4_multisignature_transaction.ts

```diff
@@ -331,4 +332,60 @@ export class MultisignatureTransaction extends BaseTransaction {
 
 		return [];
 	}
+
+	public addMultisignature(
+		_: StateStore,
```

Comment:
since we are using this variable, we should assign it a proper name.

Reply:
Good catch; I failed to renamed on the last refactor. Will rename.